tests/virt_autotest/set_config_as_glue.pm

```diff
@@ -19,12 +18,14 @@ sub fufill_guests_in_setting {
     my $vm_types = "sles|win|opensuse|oracle";
     my $get_vm_hostnames = "virsh list --all | grep -E \"${vm_types}\" | awk \'{print \$2}\'";
     my $vm_hostnames = script_output($get_vm_hostnames, $wait_script, type_command => 0, proceed_on_failure => 0);
-    my @vm_hostnames_array = split(/\n+/, $vm_hostnames);
-    foreach (@vm_hostnames_array) {
-        my $get_vm_macaddress = "virsh domiflist --domain $_ | grep -oE \"([0-9|a-z]{2}:){5}[0-9|a-z]{2}\"";
-        my $vm_macaddress = script_output($get_vm_macaddress, $wait_script, type_command => 0, proceed_on_failure => 0);
-        $virt_autotest::common::guests{$_}->{macaddress} = $vm_macaddress;
-    }
+    #    my @vm_hostnames_array = split(/\n+/, $vm_hostnames);
+    $vm_hostnames =~ s/\n+/,/g;
+    set_var("TEST_GUESTS", $vm_hostnames);
+    #    foreach (@vm_hostnames_array) {
+    #        my $get_vm_macaddress = "virsh domiflist --domain $_ | grep -oE \"([0-9|a-z]{2}:){5}[0-9|a-z]{2}\"";
+    #        my $vm_macaddress = script_output($get_vm_macaddress, $wait_script, type_command => 0, proceed_on_failure => 0);
+    #        $virt_autotest::common::guests{$_}->{macaddress} = $vm_macaddress;
+    #    }
 }
```

Comment:
We added set_config_as_clue.pm in SLE15sp5 tests only for sake of supporting MU tests by adding guest's MAC and IPs to the guest hash in the past. SLE15sp5 test itself does not need this module at all. We have a test suite setting named GUEST_PATTERN which defined what guest shall be tested.

Reply:
I quickly checked some code. set_config_as_clue.pm is to map MAC and collect VMs installed by virt_installos. If we don't want to use  set_config_as_clue. Your GUEST_PATTERN and my TEST_GUESTS have to be the same Var. 

If using my modified  set_config_as_clue in this PR, we can use the both two vars.